django/db/models/functions/base.py

```diff
@@ -201,10 +201,7 @@ def __init__(self, expression, substring, **extra):
         expression: the name of a field, or an expression returning a string
         substring: a string to find inside expression
```

Comment:
actually removed it - none of the other functions in the file have a docstring on __init__